src/_locales/hi/messages.json

```diff
@@ -1,72 +1,72 @@
 {
 	"appDesc": {
-		"message": "अपना समय बर्बाद ना करें। यूनिवर्सल बायपास आप को तंग करने वाले शॉर्टर्स लिंक से निजात दिलाता है।"
+		"message": "अनुपालना में अपना समय बर्बाद ना करें। FastForward कष्टप्रद लिंक शॉर्टंरों को रोकता है।"
```

Comment:
शॉर्टंरों is a bit hard to understand wouldn't "shorteners"(english)  be better?

Reply:
Yeah, you are right, This would be much better - 'शॉर्टनरों'